extensions/adobe/experience/adcloud/search/campaign-all.example.1.json

```diff
@@ -0,0 +1,22 @@
+{
+    "@id": "100000123",
```

Comment:
`@id` is special in JSON-LD.  You probably want to be using `xdm:id`

Reply:
Resolved by replacing with xdm:id